src/.vuepress/theme/Layout.vue

```diff
@@ -51,7 +51,43 @@ export default {
   data() {
     return {
       sidebarOpen: false,
-      sdkList
+      sdkList,
+      links: {
```

Comment:
This kind of data is not supposed to be present in the `data` of the component I think...

Reply:
Where am I supposed to put it ? :)

Reply:
config file or constant variable